modules/multi-architecture-scheduling-workloads.adoc

```diff
@@ -0,0 +1,58 @@
+// module included in the following assembly 
+//
+//post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-scheduling-workloads_{context}"]
+
+= Creating a RuntimeClass for scheduling mechanisms 
+
+Using a `RuntimeClass` object simplifies the use of scheduling mechanisms, including taints and tolerations. You can apply a runtime class to your pod and schedule them to your ARM64 or AMD64 compute node. 
+
+.Prerequisites 
+
+* You installed the OpenShift CLI (`oc`)
+* You created an ARM64 compute machine set on your cluster
+
+.Procedure 
+. Create a `RuntimeClass` object with the following command:
++
+-- 
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+.Example `RuntimeClass` object:
+[source,yaml]
+----
+apiVersion: node.k8s.io/v1beta1
+kind: RuntimeClass
+metadata:
+  name: <runtime_class_name> <1>
+handler: '' //not sure what these values are 
+scheduling:
+  nodeSelector: <2>
+    kubernetes.io/os: 'linux'
+    kubernetes.io/arch: 'arm64'
+  tolerations: 
```

Comment:
another description of the case where we need tolerations - if the node that we want the pod to land on is tainted, we add a toleration. Instead of adding it to each pod we want, we just add these constraints to the runtimeclass and just make the pods use that for easy scheduling.

Reply:
Gotcha! Yeah that makes sense, and I can add a description like that as well